Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj

```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{A263C8D9-D9AD-416A-B127-F98C84642E6A}</ProjectGuid>
```

Comment:
Check whether it exist in all other samples.

Reply:
Yes. It exists in other samples.